content/10.methods.md

```diff
@@ -9,7 +9,7 @@ We found recipients listed for the years 2009-2019.
 We gleaned the full name of the fellow as well as the year in which they received the honor.
 We used the name as provided on the site. For certain methods we were required to split the full name into first and last names.
 In this case we chose the first non-initial name as the first name and the final name as the last name.
-We did not consider a hyphen to be a name separator, so hyphenated names were both included.
+We did not consider a hyphen to be a name separator, so both side of hyphenated names were included.
```

Comment:
```suggestion
We did not consider a hyphen to be a name separator: for hyphenated names all were included.
```
"sides" feels like an odd way to refer to this...  but I'm not sure this is better 🤷‍♂

Reply:
How about "parts" or "components"? 
> We did not consider a hyphen to be a name separator: for hyphenated names, all components were included.

Reply:
👍